manpages/lumina-search.8

```diff
@@ -1,20 +1,52 @@
-.Dd October 27, 2017
+.Dd November 3, 2017
 .Dt LUMINA-SEARCH 8
 .Os  1.3.3
 
 .Sh NAME
 .Nm lumina-search
-.Nd is a simple graphical utility to search for files and applications on the system.
+.Nd a simple graphical utility to search for files and applications.
 
 .Sh SYNOPSIS
 .Nm
+.Op Fl no-excludes
+.Op Fl dir Op DIRECTORY
+.Op Fl "search \fI term"
 
 .Sh DESCRIPTION
 .Nm
-brings up the graphical utility to search the system. There are two search filters
-to chose from: "Applications" and "Files or Directories". The utility will search the
-system depending on the selected filter. Search results populate a list with
-the most relevant options first.
+brings up the graphical utility to search the system.
+There are two search filters to chose from: "Applications" and
+"Files or Directories".
+The utility will search the system depending on the selected filter.
+Search results populate a list with the most relevant options first.
+.Pp
+The following are optional commands for
+.Nm
+:
+.Bl -tag -width indent
+.It Ic -no-excludes
+Do not exclude anything from the search.
+.It Ic -dir
+Opens the utility pointed at the working directory.
+.It Ic -dir Ar DIRECTORY
+Opens the utility pointed at the specified directory.
+.It Ic -search Ar term
+Opens the utility and searchs the specified term.
+Include double quotes if searching multiple terms.
+.El
+
+.Sh EXAMPLES
+.Pp
+.Dl % lumina-search -dir /home/aaron/Downloads/
+Opens search utility pointing to Downloads.
+.Pp
+.Dl % lumina-search -search testing
+Searches applications for things relavent to the term "testing".
+
```

Comment:
There is a special case mentioned above: "Include double quotes if searching multiple terms." This would be good to add to the EXAMPLES section instead of the single term search.

Reply:
I did try that at first; However, I couldn't get the double quotes to display. I think it has something to do with the ".Dl" macro at the begging of the line. I tried to find documentation about it online as well, and I couldn't find anything. @Mrt134

Reply:
Yeah, I forgot the .Dl macro tells the manual page builder to not render anything that comes after and it probably thinks the double quotes is something it needs to render. We'd have to figure out another way to set that example up, or leave it for now.

Reply:
Oh yeah - "relavent" is misspelled ^ = "relevant"